journal/modernized/04-chapter-3.adoc

```diff
@@ -115,6 +121,8 @@ To my dear children,
 how do I feel my heart filled with paternal love to them all!--and renewedly so,
 at this time while I am writing.
 
+--
+
 +++[+++From the preceding period till the 1st of Twelfth Month,
```

Comment:
So, this narrator section does NOT need a small break _above_ because there is already a space after the embedded letter.  But you should put a small-break in _after_ this paragraph, to separate it from where the journal picks back up.